base/deprecated.jl

```diff
@@ -1913,6 +1913,9 @@ end
 @deprecate float(x::AbstractString) parse(Float64, x)
 @deprecate float(a::AbstractArray{<:AbstractString}) parse.(Float64, a)
 
+# deprecate bits to bitstring
+@deprecate bits(x) bitstring(x)
```

Comment:
I suppose this could be `@deprecate bits bitstring`?

Reply:
Good call! Simplified. Thanks! :)